src/main/java/org/hawkular/client/android/util/Uris.java

```diff
@@ -20,12 +20,21 @@
 import android.support.annotation.NonNull;
 import android.text.TextUtils;
 
+import java.io.UnsupportedEncodingException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.net.URLEncoder;
 import java.util.List;
 import java.util.Map;
 
 public final class Uris {
+    private static final class Charsets {
+        private Charsets() {
+        }
+
+        public static final String UTF_8 = "UTF-8";
```

Comment:
Wow, I would have expected that there is a constant already somewhere, but apparently not.